cea/constants.py

```diff
@@ -29,7 +29,7 @@
 
 # grey emissions
 CONVERSION_AREA_TO_FLOOR_AREA_RATIO = 1.5  # conversion component's area to floor area
-SERVICE_LIFE_OF_BUILDINGS = 60  # service life of standard building components and materials SIA2032
+SERVICE_LIFE_OF_BUILDINGS = 120  # service life of standard building components and materials SIA2032
```

Comment:
Has this value been changed in a newer version of SIA 2032? If so, could you please add the year of the new version of the standard? And if not, could you add the new source?

Note:
```
To avoid a zero result for df[total_column] in buildings over 60 years, 
SERVICE_LIFE_OF_BUILDINGS must change to 120 years in workflows/constants.py
```

This is not a good reason to change a value in the constants. If SIA2032 recommends using 60 years, then our equations should match that assumption rather than changing the constant to match our equation. If this assumption is outdated (e.g., if there is a new version of SIA2032 recommends a value of 120 years), then the reference should be updated too.

Reply:
Also, when would a value of 60 lead to a "zero result"? Since you're using `np.ceil` the value you get for `ni` is always >= 1

Reply:
The payback period in SIA 2023 /2020 is still 60 years. The longer period is suggested by literature and DUREE project (https://www.aramis.admin.ch/Texte/?ProjectID=38626).

In cea the difference between “construction year“ and “year of calculation” is the basis for calculating building GHG. If the value is larger than 60, cea assumes GHG to be Zero. 
See lines 226-228 in embodied.py.

Reply:
Yes, I'm aware that after the SERVICE_LIFE_BUILDINGS is complete CEA assumes the embodied emissions to have been "paid off". But as currently written a building of more than 120 years of age (say, the ETH Hauptgebäude, for example) will also have 0 emissions. So I don't think that's a good reason to change the value for the service life of buildings in CEA.

An updated value in literature could indeed be a good reason, but then the source should be updated in the comment as well.

Reply:
Furthermore, I've quickly gone through the DUREE documentation and I'm not sure they _recommend_ a life span of 120 years.

They mention this value as the highest found in the literature:
> The national LCA methods generally uses conventional values for this parameter. In Switzerland, the LCA national method (SIA 2032 and SIA 2040 technical books [12]) proposes 60 years. The SIA 480 standard does not define an RSP but the service life of the building structure instead. The SIA 480:2004 standard [41] considers from 80 to 100 years while the revised 2016 version considers from 40 to 120 years with an intermediate value at 75 years [42]. In addition, the SNARC method, used in early design stages, considers 30 years [43]. Other LCA methods in Europe consider 50 years [44], 80 years [45] or even 120 years [46].

And then they considered a range of possible lifespans:
> Except for the baseline study, the influence of different methodological choices on the LCA and LCC result was studied. In the beginning, the RSP was varied from 30 to 120 years and the results showed that the contributions of the LCA and LCC stages change as a function of the chosen RSP, as well as the magnitude of the Sobol’ Indices. 

And in their recommendations for practitioners they seem to recommend such a scenario-based approach:
> The building RSP is an influential parameter on the LCA and LCC uncertainty. Different scenario analyses are recommended to capture more accurately the variability of the LCA and LCC.

Reply:
Ok, looks like the value was reverted to 60. I'm resolving this conversation now.